CONTRIBUTING.md

```diff
@@ -70,6 +71,10 @@ Please *do not* issue a PR containing a new plan for the specific version of sof
 
 Issues with an associated gist containing a working fork of our current plan which builds the version of the software will be attended to first.
 
+## Dependencies
+
+If your package has dependencies, it should only depend on other packages in the core origin.
```

Comment:
i think that `should` is a `must` (in the context of https://www.ietf.org/rfc/rfc2119.txt)

Reply:
As the spouse of a 2nd year law student I can appreciate this specification, taken care of.

Reply:
I might also suggest a note here that says something along the lines of: If a dependency you need does not exist in core, please open a pull-request to add the dependency or open an issue requesting the dependency be added by a core-maintainer.